### PR TITLE
Create Python package for AppTaskQueue

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1754/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1760/

File: AppController/lib/app_manager_client.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ class AppManagerClient
   attr_reader :conn, :ip
 
   # The port that the AppManager binds to, by default.
-  SERVER_PORT = 49934
+  SERVER_PORT = 17445
```
Comment (menivaitsi) [inline AppController/lib/app_manager_client.rb]: Will the existing monit config file (suffix: -49934.cfg) be replaced upon down/up.?

File: AppController/lib/taskqueue.rb [hunk 2]
```diff
@@ -37,8 +37,7 @@ module TaskQueue
   # The location of the taskqueue server script. This service controls 
   # and creates celery workers, and receives taskqueue protocol buffers
   # from AppServers.
-  TASKQUEUE_SERVER_SCRIPT = File.dirname(__FILE__) + "/../../AppTaskQueue" + \
-                            "taskqueue_server.py"
+  TASKQUEUE_SERVER_SCRIPT = '/usr/local/bin/appscale-taskqueue'
```
Comment (obino) [inline AppController/lib/taskqueue.rb]: Did I miss when/where we install the script there?

File: AppController/lib/app_manager_client.rb (hunk 1 above)
Comment (cdonati) [inline AppController/lib/app_manager_client.rb]: Yes, the conf.d directory is cleared on a down.

File: AppController/lib/taskqueue.rb (hunk 2 above)
Comment (cdonati) [inline AppController/lib/taskqueue.rb]: It gets created during the build. (`python setup.py install` inside the AppTaskQueue directory). I wanted to do a `which appscale-taskqueue` here, but /usr/local/bin isn't on the path for init scripts unless we wanted to change it explicitly there.

**Pull request merged**